### ISSUE TITLE
ImportError: cannot import name 'bilinear_sample_nchw'

### ISSUE THREAD

**Issue by apivovarov** [apache/incubator-tvm#4371]
Same issue, tvm, nnvm and topi were installed after the build
```
~/tvm/python$ sudo python3 setup.py install
~/tvm/nnvm/python$ sudo python3 setup.py install
~/tvm/topi/python$ sudo python3 setup.py install
```

But model compilation fails
```
Traceback (most recent call last):

  File "./compile.py", line 84, in <module>
    from tvm import relay

  File "/usr/local/lib/python3.6/dist-packages/tvm-0.6.dev0-py3.6-linux-x86_64.egg/tvm/relay/__init__.py", line 27, in <module>
    from . import expr_functor

  File "/usr/local/lib/python3.6/dist-packages/tvm-0.6.dev0-py3.6-linux-x86_64.egg/tvm/relay/expr_functor.py", line 24, in <module>
    from .op import Op

  File "/usr/local/lib/python3.6/dist-packages/tvm-0.6.dev0-py3.6-linux-x86_64.egg/tvm/relay/op/__init__.py", line 20, in <module>
    from .op import get, register, register_schedule, register_compute, register_gradient, \

  File "/usr/local/lib/python3.6/dist-packages/tvm-0.6.dev0-py3.6-linux-x86_64.egg/tvm/relay/op/op.py", line 19, in <module>
    import topi

  File "/usr/local/lib/python3.6/dist-packages/topi-0.6.dev0-py3.6.egg/topi/__init__.py", line 43, in <module>
    from . import nn

  File "/usr/local/lib/python3.6/dist-packages/topi-0.6.dev0-py3.6.egg/topi/nn/__init__.py", line 23, in <module>
    from .deformable_conv2d import *

  File "/usr/local/lib/python3.6/dist-packages/topi-0.6.dev0-py3.6.egg/topi/nn/deformable_conv2d.py", line 23, in <module>
    from ..cpp.image import bilinear_sample_nchw

ImportError: cannot import name 'bilinear_sample_nchw'
```

If tvm, nnvm and topi were installed using setup.py we do not need to set `LD_LIBRARY_PATH`

Anyone looking at the issue?

Discussion: https://discuss.tvm.ai/t/cannot-import-name-bilinear-sample-nchw/2510

**Comment by OriAlpha** [comment]
It's just a broken path error, you can do 

export LD_LIBRARY_PATH=/home/suhas/.local/lib/python3.6/site-packages/topi-0.6.dev0-py3.6.egg/topi

**Comment by tqchen** [comment]
Thanks @OriAlpha @apivovarov please  use the discuss forum for future trouble shooting

**Comment by apivovarov** [comment]
@OriAlpha You do not understand. If I install `tvm`, `nnvm` and `topi` using `sudo python3 setup.py install` I do not have to set anything to `LD_LIBRARY_PATH`.
But currently I have to do it. It is not right.

@ tqchen This is obviously an issue

**Comment by kalcohol** [comment]
> @OriAlpha If I install `tvm`, `nnvm` and `topi` using `sudo python3 setup.py install` I do not have to set anything to `LD_LIBRARY_PATH`.
> But currently I have to do it. It is not right.
> 
> @tqchen This is obviously an issue

yes, i miss this too in windows 10. the temp way that setting LD_LIBRARY_PATH could avoid this too(set PATH instead).

but i agree with u, it's an issue, should be fix, should not be closed easily.

releated issue: #3162 #4361 
as a new to python, i do not know how to fix this in python's way directly, so i cannot pr a fix.

**Comment by kalcohol** [comment]
i know what is the mechanism of the occurrence, topi depends on tvm module, tvm module import this missing function for topi, but tvm itself can not find libtvm_topi.so or tvm_topi.dll, so when importing tvm.relay or topi directly, error occurs.

but no pr idea now, as before.

**Comment by murez** [comment]
well, it may be works under linux but in my windows env changing the system path setting does not work even after reboot. Can any one fix the problem.

**Comment by poincarelee** [comment]
why closed this issue? I don't think you really solve it. 
I did not find bilinear_sample_nchw in any folder so I replaced it with the resize module, however, when I run the main.py, a new error occurs: ValueError: Compute is already registered in autoTVM task conv2d_NCHWc.x86.
I am not sure whether this is a little problem, while closing the issue easily is really a stupid choice.

**Comment by junrushao1994** [comment]
The original issue was closed because it had been fixed. Discussion thread: https://discuss.tvm.ai/t/cannot-import-name-bilinear-sample-nchw/2510/30?u=junrushao1994.

@poincarelee if it is still a problem for you, this would be super helpful for the community if you ask on the discussion forum. Thanks!

**Comment by poincarelee** [comment]
Thanks a lot, this problem was solved indeed. I retried and it worked. Thanks again.
















At 2020-05-25 09:43:09, "Junru Shao" <notifications@github.com> wrote:

The original issue was closed because it had been fixed. Discussion thread: https://discuss.tvm.ai/t/cannot-import-name-bilinear-sample-nchw/2510/30?u=junrushao1994.

@poincarelee if it is still a problem for you, this would be super helpful for the community if you ask on the discussion forum. Thanks!
